app/controllers/media_controller.rb

```diff
@@ -56,18 +59,13 @@ def download
     end
   end
 
-protected
-
-  def has_bypass_id_for_asset?(asset)
-    token = params.fetch(:token, cookies[:auth_bypass_token])
-    asset.valid_auth_bypass_token?(token)
-  end
-
   def authorized_for_asset?(asset)
     return true unless requested_from_draft_assets_host?
 
     return true if has_bypass_id_for_asset?(asset)
 
+    return true if draft_asset_manager_access? && asset.access_limited? == false
```

Comment:
You might want to change this to: `return true if draft_asset_manager_access? && !asset.access_limited?`